src/reducers/weatherReducer.tsx

```diff
@@ -0,0 +1,21 @@
+import { Fetch_Weather } from "../action/constant";
+
+const initialstate = {
+  days: [],
+  current: [],
+  weather: [],
+};
+
+export default function (state = initialstate, action: any) {
+  switch (action.type) {
+    case Fetch_Weather:
```

Comment:
action types should be uppercase

```suggestion
    case FETCH_WEATHER:
```